qa-tests/test_script.bat

```diff
@@ -80,7 +80,7 @@ GOTO ARG_LOOP
 
 
 
-REM As this is a bash script it is intended to be executed on Linux Plarforms.
+REM As this is a bash script it is intended to be executed on Windows Platforms.
```

Comment:
Hi , I think initial comment was correct here

Sébastien

Reply:
Hi,
in the next line it says `test_directory=ctest_suite_win64`
also to my knowledge `bat` scripts are used only on win

But I am not a win expert, I can revert

Reply:
Hi , ok , I missed the name of the script , in this case this should be '**bat** script it is intended to be executed on **Windows Platforms**'

Thanks,
Sébastien